### PR TITLE
Create application process file 

### PR DESCRIPTION
**Pull request by blackinno**

### CODE REVIEW

**Comment by antho1404** [comment]
Please rename applocation in application ;)

File: README.md [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-./start
+
+## Specify an ERC20
+
+By default this service emits event from any ERC20. To specify one, set the CONTRACT_ADDRESS env variable to the address of the ERC20 to only listen to.
+
+By example, to listen to only the MESG Token, add to the deploy command:
+
+```bash
+mesg-cli process:dev application.yml --env PROVIDER_ENDPOINT=https://mainnet.infura.io/v3/PROJECT_ID --env CONTRACT_ADDRESS=0x...
```
Comment (antho1404) [inline README.md]: This doesn't work

File: applocation.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Erc20-analytics
+steps:
+  - type: trigger
+    instance:
+      src: https://github.com/mesg-foundation/service-ethereum-erc20
+      env:
+        - PROVIDER_ENDPOINT=$(env:PROVIDER_ENDPOINT)
+        - CONTRACT_ADDRESS=
```
Comment (antho1404) [inline applocation.yml]:
```suggestion
        - CONTRACT_ADDRESS=$(env:CONTRACT_ADDRESS)
```

File: applocation.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          to: transferData.to,
+          value: parseFloat(transferData.value),
+          contractAddress: transferData.contractAddress
+        })
+      inputs:
+        blockNumber: { key: blockNumber }
+        contractAddress: { key: contractAddress }
+        from: { key: from }
+        to: { key: to }
+        value: { key: value }
```
Comment (antho1404) [inline applocation.yml]: I think this whole task is unnecessary. You can directly access the data from the trigger in the `influxdb#write` tasks

**Review by antho1404 — CHANGES_REQUESTED** [review]

File: applocation.yml (hunk 3 above)
Comment (blackinno) [reply applocation.yml]: the task `js-function` is using for convert value from `string` to `float`. The value data can save value as a string at `influxdb`?

Comment (antho1404) [reply applocation.yml]: could you try to save it as string, maybe influxdb is automatically doing the conversion. In that case, we could remove this task, otherwsie, it's good like that.

Comment (blackinno) [reply applocation.yml]: I test to save it as a string, the influxdb isn't doing the conversion so the Grafana isn't displayed any data.

Grafana dashboard:
Screen Shot 2562-12-24 at 14 36 41

process log: 
Screen Shot 2562-12-24 at 14 37 04

**Review by antho1404 — APPROVED** [review]

**Pull request merged**